README.md

```diff
@@ -1,85 +1,135 @@
+full list of options please see `tar_scm.service.in`.
 
-Apart from various SCM like git, hg, bzr or svn it additionally
+Apart from various SCM like git, hg, bzr or svn, it additionally
 supports `--url` option that allows you to specify URL of the upstream
 tarball to be downloaded.
 
-This is the git repository for
-[openSUSE:Tools/obs-service-tar_scm](https://build.opensuse.org/package/show/openSUSE:Tools/obs-service-tar_scm).
-The authoritative source is
-https://github.com/openSUSE/obs-service-tar_scm
-
-The service can be used in combination with other services like
+`tar_scm` can be used in combination with other services like
 [download_files](https://github.com/openSUSE/obs-service-download_files),
 [extract_file](https://github.com/openSUSE/obs-service-extract_file),
 [recompress](https://github.com/openSUSE/obs-service-recompress) or
 [set_version](https://github.com/openSUSE/obs-service-set_version)
 e.g. within the [GIT integration](https://en.opensuse.org/openSUSE:Build_Service_Concept_SourceService#Example_2:_GIT_integration)
 workflow.
 
-## Archive Formats
+**`tar_scm` is deprecated in favour of `obs_scm`.**
 
-### obscpio
+### obs_scm
 
-The `obscpio` archive format is the new OBS specific archive format
-based on `cpio` which can be consumed and generated by the [OBS Delta
-Store](http://openbuildservice.org/help/manuals/obs-reference-guide/cha.obs.architecture.html#delta_store)
-to store changes in an incremental way independently from your chosen
-SCM.  The `obs_scm` service generates archives in this format, which
-is basically the `cpio` `newc` format.
+`obs_scm` is similar in concept to `tar_scm`, but instead of directly
+generating tarballs, it instead uses the new `obscpio` archive format
+(see below) as an intermediate space-efficient format in which to
+store the sources.
+
+**It is recommended to use `obs_scm` in favour to `tar_scm`**, because
+it helps to save a lot of disk space on the server side, especially
+when used in continuous integration (e.g. nightly builds and builds of
+pull requests).
+
+The usual source tarballs can be regenerated from this at build-time
+using the `tar` and `recompress` source services, so no changes to
+`.spec` files are required when switching to `obs_scm` and `obscpio`.
+Having said that, it may be more efficient to drop the build-time
+usage of `recompress`, since at build-time `rpmbuild` would decompress
+the same file soon after compressing it.  The decision whether to use
+`recompress` will depend on what format is desired within the
+resulting `.src.rpm`, and on the types of compression being used for
+both the tarball and by `rpmbuild` for constructing the source rpms.
+
+`obs_scm` additionally generates a file named `<package>.obsinfo`
+which includes useful information from your SCM system, such as the
+name, version number, mtime, and commit SHA1.  This data is then used
+by the `tar` service (see below) to reconstruct a tarball for use by
```

Comment:
and also by the set_version service to patch the build description files.

Reply:
Oh nice, I didn't realise that - thanks!

Reply:
Fixing this in the next push ...